python/tests/expressions/test_visitors.py

```diff
@@ -836,15 +843,23 @@ def _create_manifest_evaluator(bound_expr: BoundPredicate) -> _ManifestEvalVisit
     return evaluator
 
 
-def test_manifest_evaluator_less_than_no_overlap():
-    expr = BoundLessThan(
-        term=BoundReference(
-            field=NestedField(field_id=1, name="foo", field_type=StringType(), required=False),
-            accessor=Accessor(position=0, inner=None),
-        ),
-        literal=StringLiteral("c"),
-    )
+@pytest.fixture
+def string_schema() -> Schema:
+    return Schema(NestedField(field_id=1, name="foo", field_type=StringType(), required=False))
```

Comment:
Nit: I'd prefer using better names that make the tests more readable, like like `str_col` because these are in a separate file.

Reply:
Ah, I like that a lot. You can see the type from which fixture is being used, but more explicit is more better indeed. Thanks!